components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/UserAwareAPIConsumer.java

```diff
@@ -134,17 +134,9 @@ public API getAPI(APIIdentifier identifier) throws APIManagementException {
     }
     
     @Override
-    public ApiTypeWrapper getAPIorAPIProductByUUID(String uuid, String requestedTenantDomain)
+    public ApiTypeWrapper getAPIorAPIProductByUUID(String uuid, String orgId)
             throws APIManagementException {
-        ApiTypeWrapper apiTypeWrapper = super.getAPIorAPIProductByUUID(uuid, requestedTenantDomain);
-        Identifier identifier;
-        if (apiTypeWrapper.isAPIProduct()) {
-            identifier = apiTypeWrapper.getApiProduct().getId();
-        } else {
-            identifier = apiTypeWrapper.getApi().getId();
-        }
-        //TO-DO Commented to since need to work for monodb should be move to registry persistent impl
-//        checkAccessControlPermission(identifier);
+        ApiTypeWrapper apiTypeWrapper = super.getAPIorAPIProductByUUID(uuid, orgId);
```

Comment:
Is this change necessary?

Reply:
That identifier object is not used anywhere. Thus removed it from the implementation.